packages/workflow/src/Workflow.ts

```diff
@@ -1228,7 +1228,6 @@ export class Workflow {
 
 	/**
 	 * Executes the given node.
-	 *
```

Comment:
were you working on passing down the AbortSignal to executions?

Reply:
No, just something I noticed when going thru the code. I created a follow-up for this: https://linear.app/n8n/issue/ADO-1600/bug-prevent-new-executions-when-app-is-shutting-down Feel free to grab it if you want